src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2128,12 +2127,12 @@ static HttpStatusCode mongoUpdateCasubNewNotification(std::string subId, std::st
 */
 bool processAvailabilitySubscription
 (
-  EntityIdVector  enV,
-  AttributeList   attrL,
-  std::string     subId,
-  std::string     notifyUrl,
-  Format          format,
-  std::string     tenant
+  EntityIdVector     enV,
+  AttributeList      attrL,
```

Comment:
Of course... but that is part of a next PR. The "strategy" is going PR by PR with small steps (due to each change involves many files).

NTC